database/model.go

```diff
@@ -87,6 +93,27 @@ func (config *InstanceConfig) WaitBetweenRetries() time.Duration {
 	return config.ConnectRetryWait
 }
 
+// NumberOfDeltaLockTries on a connection to the database before failing
+func (config *InstanceConfig) NumberOfDeltaLockTries() int {
+	return config.DeltaLockMaxTries
```

Comment:
default this to like 10 if '<= 0' no retries would not make sense in this context